### PR TITLE
Profile style

### PR DESCRIPTION
**Pull request by NicoLorenzo1**
Added: 
css styles on post-card component and others

### CODE REVIEW

File: patitasuy/src/app/components/post-card/post-card.component.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
-<mat-card routerLink="/details/{{post.id}}">
-  <img mat-card-image src="{{post.url}}" alt="Pet image">
+<mat-card class="custom-card" routerLink="/details/{{post.id}}">
+  <img class="image" src="{{post.url}}" alt="Pet image">
   <mat-card-content>
     <p>{{post.title}}</p>
     <p>{{post.location}}</p>
+    <p class="state">{{post.state}}</p>
```
Comment (danacastroman) [inline patitasuy/src/app/components/post-card/post-card.component.html]: Make sure the status is only being displayed on the profile view, home view should not display the status

**Review by danacastroman — CHANGES_REQUESTED** [review]

File: patitasuy/src/app/components/post-card/post-card.component.ts [hunk 2]
```diff
@@ -1,16 +1,29 @@
...
-export class PostCardComponent  implements OnInit {
+export class PostCardComponent implements OnInit {
 
   @Input() post!: Post;
 
-  constructor() { }
+  constructor(public userService: UserService) { }
 
-  ngOnInit() {}
+  getCurrentUserId() {
```
Comment (alfonsoLeandro) [inline patitasuy/src/app/components/post-card/post-card.component.ts:17]: This method will always return 0, it needs to be async, or be executed once in the constructor

File: patitasuy/src/app/pages/profile/profile.component.scss [hunk 3]
```diff
@@ -81,16 +63,10 @@
...
-.descriptionState {
-    font-family: 'Roboto';
-    font-style: normal;
-    font-size: 1em;
-    color: #FF9934;
-}
-
 .editButton {
-    --background: #FF9934;
+    --background: rgb(31, 154, 154);
```
Comment (alfonsoLeandro) [inline patitasuy/src/app/pages/profile/profile.component.scss:67]: Replace this with color="secondary" attribute in the button with the editButton class

File: server/routes/post.routes.ts [hunk 4]
```diff
@@ -5,10 +5,11 @@ import validateJWT from "../middlewares/validate-jwt";
 const router = Router();
 
 router.get('/', [validateJWT], getPosts);
-router.get('/details/:postId', [validateJWT,existsPost], getPost);
+router.get('/details/:postId', [validateJWT, existsPost], getPost);
 router.get('/my-user/:userId', [validateJWT], getMyPosts);
```
Comment (alfonsoLeandro) [inline server/routes/post.routes.ts:9]: This endpoint is duplicated. Remove this one

File: server/controllers/post.controller.ts
Comment (alfonsoLeandro) [inline server/controllers/post.controller.ts:1]: Change indent size to 4

**Pull request closed without merging**